torch_np/_wrapper.py

```diff
@@ -4,14 +4,22 @@
 pytorch tensors.
 """
 
+from typing import Optional, Sequence
+
 import torch
 
-from . import _funcs
+from . import _decorators, _dtypes, _funcs, _helpers
 from ._detail import _dtypes_impl, _flips, _reductions, _util
 from ._detail import implementations as _impl
-from ._ndarray import array, asarray, asarray_replacer, maybe_set_base, ndarray, newaxis
-
-from . import _dtypes, _helpers, _decorators  # isort: skip  # XXX
+from ._ndarray import array, asarray, maybe_set_base, ndarray
+from ._normalizations import (
```

Comment:
What's the difference between `_funcs.py` and `_wrapper.py`? Should we merge them?

Reply:
Absolutely. The split is temporary, and we'll cleanly merge them once the cleanups from other PRs further up the stack are done. 
At this stage, imports from ._ndarray cause circular imports. So either let's live with the split for a while, or I can bloat this PR with what's up the stack. Reviewer's choice :-).